spec/index.html

```diff
@@ -94,7 +94,13 @@
 
     }
     .ex th { text-align: center; }
-  </style>
+
+    table.cp-definitions { border-collapse:collapse; background-color: #DDDDFF}
+
+    table.cp-definitions,
+    table.cp-definitions th,
+    table.cp-definitions td { border:1px solid black; padding:0.2em; }
```

Comment:
I don't see a quick way to do this selectively ... but I'd like the Glyph column to be center-aligned, while the others do make sense left-aligned.

Reply:
Done in https://github.com/w3c/rdf-turtle/pull/46/commits/2c0d68f7ae4336eaa95610e77d412be29ba8908e.